crates/rolldown/src/stages/generate_stage/code_splitting.rs

```diff
@@ -80,6 +90,13 @@ impl<'a> GenerateStage<'a> {
       FxHashMap::with_capacity(self.link_output.entries.len());
     // Create chunk for each static and dynamic entry
     for (entry_index, entry_point) in self.link_output.entries.iter().enumerate() {
+      // IIFE format should inline dynamic imports
```

Comment:
Since dynamic imported modules are inlined, why there still are dynamic entries? Should we just not create dynamic entry in `module_loader`?

Reply:
Good idea. Done.